tool/csv_writer_gobuster.go

```diff
@@ -0,0 +1,67 @@
+package tool
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+)
+
+func remove(slice []string, s int) []string {
+	return append(slice[:s], slice[s+1:]...)
+}
+
+func changePrefix(line string) string {
+	rUp, _ := regexp.Compile(`+\[[0-9]+m([A-Z]+)++\[0m`)
```

Comment:
I can't because there is special char ESC
![image](https://user-images.githubusercontent.com/60828505/179392627-4d5eb5d6-7eea-4133-8b28-a40961b740a8.png)